### PR TITLE
Server list files contribute RPC URLs to specific blockchains (either MainNet, TestNet, unknown private chain or named private chain); remember last used RPC server (per-blockchain); different icons for different blockchain types

### LINKED ISSUE #65: Different logo for neo express instances in blockchain explorer.
I've reached out to Neo marketing for a logo. In the meantime, there are a bunch of SVGs from the neo.org website [available](https://github.com/neo-project/neo.org/blob/master/NeoWeb/wwwroot/images/). In particular I thought the icon from inside the silhouette head of [CoreDeveloper](https://github.com/neo-project/neo.org/blob/master/NeoWeb/wwwroot/images/dev/CoreDeveloper.svg), [ToolsDeveloper](https://github.com/neo-project/neo.org/blob/master/NeoWeb/wwwroot/images/dev/ToolsDeveloper.svg) or [dApp Developer](https://github.com/neo-project/neo.org/blob/master/NeoWeb/wwwroot/images/dev/dAppDeveloper.svg) would work

### LINKED ISSUE #79: Support accessing arbitrary neo blockchains from Visual DevTracker

### LINKED ISSUE #80: Add Default or Last Used RPC node when executing operations directly on blockchain
currently, if you execute an operation like claim or transfer directly on a blockchain instance, it asks you which RPC node to use every time. There should be a mechanism for devtracker to remember previously used node instead of asking the user every time.

THis could be done via a "default node" mechanism or by remembering the most recently used RPC node

### LINKED ISSUE #81: mechanism to add additional RPC nodes to MainNet/TestNet blockchain
None of the CoZ testnet servers appear to support the getunspents RPC method, but those are the only ones that get populated. We need a way for users to specify their own - potentially custom - RPC nodes for mainnet and testnet (and arbitrary NEO blockchains in the future).

RPC nodes added this way would need to have their genesis block hash validated against at least one other existing RPC node for the blockchain.

Custom RPC nodes could be stored in a variety of places:
* ExtensionContext.globalState
* ExtensionContext.workspaceState
* a JSON document with custom extension in the workspace

### LINKED ISSUE #82: remove server lists feature
replaced with #81 and #79

### PR DESCRIPTION
**Pull request by djnicholson**
Fixes #82 
Fixes #81 
Fixes #79 
Fixes #65 

Related to #80

### CODE REVIEW

File: sample-workspace/neo-servers.json [hunk 1]
```diff
@@ -1,14 +1,12 @@
...
-        "name": "Neo Test Net",
-        "url": "https://test3.cityofzion.io"
-    },
-    {
-        "name": "Neo Test Net - getunspents supported",
-        "url": "http://seed5.ngd.network:20332"
+{
+    "neo-rpc-uris": [
+        "http://seed2.ngd.network:10332",
+        "https://test3.cityofzion.io",
```
Comment (devhawk) [inline sample-workspace/neo-servers.json]: test3.cityofzio.io. shows up twice now. SHould we hide one?

**Comment by devhawk** [comment]
remembering the last RPC server used feature doesn't appear to work. Everything else looks good. I'm cool with removing that this should close #80

**Comment by djnicholson** [comment]
> remembering the last RPC server used feature doesn't appear to work. Everything else looks good. I'm cool with removing that this should close #80

I thought it did.  Will leave #80 open for now, test again and ping you for repro information if needed...

File: sample-workspace/neo-servers.json (hunk 1 above)
Comment (djnicholson) [reply sample-workspace/neo-servers.json]: Done. I added some others that are not returned by the API instead.

**Pull request merged**